rocAL/rocAL/source/loaders/image/image_read_and_decode.cpp

```diff
@@ -157,7 +183,8 @@ ImageReadAndDecode::load(unsigned char* buff,
     const unsigned output_planes = std::get<1>(ret);
     const bool keep_original = decoder_keep_original;
     const size_t image_size = max_decoded_width * max_decoded_height * output_planes * sizeof(unsigned char);
-
+    bool is_external_source = (_reader_type == StorageType::EXTERNAL_FILE_SOURCE);
+    bool skip_decode = false;
```

Comment:
we have hardcoded skip_decode = false here
Please check if this is expected

Reply:
This is expected